src/Illuminate/Support/Str.php

```diff
@@ -401,9 +404,13 @@ public static function words($value, $words = 100, $end = '...')
      */
     public static function markdown($string, array $options = [])
     {
-        $converter = new GithubFlavoredMarkdownConverter($options);
+        $environment = new Environment($options);
+        $environment->addExtension(new CommonMarkCoreExtension());
+        $environment->addExtension(new GithubFlavoredMarkdownExtension());
 
-        return $converter->convertToHtml($string);
+        $converter = new MarkdownConverter($environment);
+
```

Comment:
You could revert these changes since the `GithubFlavoredMarkdownConverter` will continue to work as-is, and you're not adding any additional extensions here.  But there's no reason you can't take this more verbose route either :)

Reply:
Done.